test/src/test/java/org/corfudb/integration/CorfuStorePerfIT.java

```diff
@@ -0,0 +1,145 @@
+package org.corfudb.integration;
+
+
+import java.lang.reflect.InvocationTargetException;
+import java.nio.file.FileStore;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import org.apache.commons.io.FileUtils;
+
+import org.junit.jupiter.api.Test;
+
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.CorfuStoreMetadata;
+import org.corfudb.runtime.collections.CorfuStore;
+import org.corfudb.runtime.collections.CorfuTable;
+import org.corfudb.runtime.collections.Table;
+import org.corfudb.runtime.collections.TableOptions;
+import org.corfudb.runtime.collections.TxBuilder;
+import org.corfudb.test.SampleSchema;
+import org.corfudb.test.SampleSchema.ManagedResources;
+import org.corfudb.test.SampleSchema.Uuid;
+import org.corfudb.test.SampleSchema.EventInfo;
+
+public class CorfuStorePerfIT extends  AbstractIT {
+
+    @Test
+    public void corfuStorePerfComparisonTest() throws Exception {
+        long maxLogSize = FileUtils.ONE_MB / 2;
+        Process server1 = runServerWithQuota(DEFAULT_PORT, maxLogSize,
+            true);
+
+        CorfuRuntime rt = new CorfuRuntime(DEFAULT_ENDPOINT).connect();
+        final int count = 100;
+        addObjectsToTable(rt, count);
+        addProtoToStore(rt, count);
+    }
+
+    private void addObjectsToTable(CorfuRuntime rt, final int count) {
+        System.out.println("Start Writing Java Obj" + System.currentTimeMillis());
+        long start = System.currentTimeMillis();
+        Map<UUID, Event> map = rt.getObjectsView()
+            .build()
+            .setStreamName("s1")
+            .setType(CorfuTable.class)
+            .open();
+
+        Event event;
+        for(int i=0; i<count; i++) {
+            event = new Event(i, "TestEvent"+i, 8000,
+                20000, 10
```

Comment:
maybe use variable or pass in as parameters instead of using magic numbers.

Reply:
done. thanks